pages/welcome.js

```diff
@@ -0,0 +1,124 @@
+import Link from '../components/Link';
+import Page from '../components/Page';
+import StyledCard from '../components/StyledCard';
+import StyledLink from '../components/StyledLink';
+import { useUser } from '../components/UserProvider';
+
+import Custom404 from './404';
+
+const WelcomeOptionContainer = styled(Container)`
+  &:hover {
+    background-color: #f5faff;
+  }
+`;
+
+const Welcome = () => {
+  const { LoggedInUser, loadingLoggedInUser } = useUser();
+  if (!loadingLoggedInUser && !LoggedInUser) {
+    return <Custom404 />;
+  }
+  return (
+    <Page title="Welcome to Open Collective!">
+      <Flex flexDirection={['column', 'row']} mb="61px" mt="112px" justifyContent="center" alignItems="center">
+        <Flex textAlign="center" flexDirection="column" pr={[0, '48px']}>
+          <Box>
+            <Image src="/static/images/oc-logo-watercolor-256.png" alt="Open Collective logo" height={96} width={96} />
+          </Box>
+          <Container pt="40px" pl="16px" pr={['16px', 0]} width={['100%', '404px']}>
+            <Flex fontSize="32px" fontWeight="700" color="black.900" lineHeight="40px">
+              <FormattedMessage defaultMessage="Welcome to Open Collective!" />
+            </Flex>
+            <Flex fontSize="18px" fontWeight="400" color="black.800" lineHeight="26px" pt="14px">
+              <FormattedMessage defaultMessage="Now that you have created your personal account, there are a couple of things you can do from here..." />
+            </Flex>
+          </Container>
+        </Flex>
+        <StyledCard
+          width={['100%', '520px']}
+          display="flex"
+          flexDirection="column"
+          alignItems="center"
+          style={{ overflow: 'visible' }}
+          mt={['100px', 0]}
+        >
+          <Box mt="-64px">
+            <Image src="/static/images/sample-avatar.png" height="128px" width="128px" />
+          </Box>
+          <Flex fontSize="24px" fontWeight="700" color="black.900" lineHeight="32px" pt="40px" pb="40px">
+            {LoggedInUser?.collective?.name}
+          </Flex>
+          <Link href="/create">
+            <WelcomeOptionContainer mt="16px" width={['300px', '472px']} borderRadius="8px">
+              <Flex alignItems="center" p="13px">
+                <Box width="386px">
+                  <Flex fontSize="18px" fontWeight="700" color="black.900" lineHeight="26px">
+                    <FormattedMessage id="collective.create" defaultMessage="Create Collective" />
+                  </Flex>
+                  <Flex fontSize="15px" fontWeight="500" color="black.700" lineHeight="22px" pt="14px">
+                    <FormattedMessage defaultMessage="Create a Collective to be able to accept donations, apply for grants, and manage your budget transparently." />
+                  </Flex>
+                </Box>
+                <Box pl="39px">
+                  <Image src="/static/images/right-arrow.png" alt="Right Arrow" width="22px" height="20px" />
```

Comment:
In most places we use `→` rather than an image. Unless there are good arguments in favor of using an image, I would strive for consistency

Reply:
Yes, I thought about this, and tried this `→` but decided against this simply because looking at the design carefully to me the image looked much better. Note that the image looks rich with a slight blue color and also it has larger line size. When I added this symbol I couldn't get the same look and feel. However it's probably personal preference and I am okay with using the symbol instead if that's the consensus. 

@Betree @opencollective/design